client/src/redux/reducers/viewOptionsReducer.js

```diff
@@ -1,4 +1,5 @@
-import _ from 'lodash';
+import _findIndex from 'lodash/findIndex';
+import _get from 'lodash/get';
```

Comment:
using `lodash-es` will shake and only pull in what you use from the lodash lib.

Reply:
Up for iterating on this, we'll give the `-es` stuff a shot, sounds cleaner to use